docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,754 @@
+.. _vQuanta SKU definition:  https://github.com/RackHD/RackHD/blob/master/example/samples/vQuanta_d51_sku.json
+
+When you add a SKU, the system will check all existing nodes against the definition for that SKU and update
+the nodes to assign the SKU where it's relevant. If the SKU definitions includes any default workflows, those
+will **not** get automatically invoked when you create the SKU definition. The default workflow path will
+only operate when a node is first being discovered; or more specifically correct when the
+``Graph.SKU.Discovery`` workflow is run against the node.
+
+
+.. container:: clearer
+
+   .. image :: ../_static/invisible.png
+
+.. warning:: **SLOW ON VAGRANT**
+
+    The simulated hardware is a virtual machine inside another virtual machine, so
+    while this process works, it is very slow on most desktops. To see it operational
+    you will want probably want watch the console on the simulated hardware using VNC.
+
+    On a Macbook Pro (2.2 GHz Intel Core i7) with 16GB of ram, this process takes
+    approximately 2 hours to fully complete. Using real hardware, the process is in minutes.
+
+Invoking a workflow
+--------------------
+
+Almost all the workflows you'll want to invoke start with controlling the node
+remotely, most commonly to tell the node to reboot and start a PXE boot process. The
+simplest possible workflows just power off or power on a node. By default a node
+will not have any OBM settings defined.
+
+
+Checking and setting the OBM settings for a node
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can check to see if any OBM settings are defined on the node using the nodes
+API:
+
+``curl http://localhost:9090/api/2.0/nodes/57990efa0d76e7c207cdfc3f | jq``
+
+.. code-block:: JSON
+
+    {
+      "autoDiscover": "false",
+      "id": "57990efa0d76e7c207cdfc3f",
+      "name": "52:54:be:ef:40:98",
+      "identifiers": [
+        "52:54:be:ef:40:98"
+      ],
+      "tags": [],
+      "obms": [],
+      "type": "compute"
+    }
+
+If the node has an OBM service, the key ``obms`` will have some data in it:
+
+.. code-block:: JSON
+
+    {
+      "autoDiscover": "false",
+      "id": "57990efa0d76e7c207cdfc3f",
+      "name": "52:54:be:ef:40:98",
+      "identifiers": [
+        "52:54:be:ef:40:98"
+      ],
+      "tags": [],
+      "obms": [
+        {
+          "service": "ipmi-obm-service",
+          "ref": "/api/2.0/obms/5799101e95d9a2bf0780128a"
+        }
+      ],
+      "type": "compute"
+    }
+
+If the node does not have any OBM settings, you will want to provide one - as nearly all
+workflows start by utilizing the OBM settings. When you're creating an OBM setting
+via the API, you'll need to provide both the node ID and a relevant "host" setting for
+accesing the out of band management interface.
+
+For the node in the examples above, that could be:
+
+.. code-block:: REST
+
```

Comment:
@keedya when you have a few moments, I'd love your eyes on this - want to make sure I'm stressing the correct way to add an OBM settings value manually via the API here.

In particular, is a mac-address still acceptable as a "host" value here?